specification/cognitiveservices/data-plane/Face/v1.0/Face.json

```diff
@@ -1,2636 +1,2978 @@
+                            "$ref": "#/definitions/APIError"
+                        }
+                    }
+                },
+                "produces": [
+                    "application/json"
+                ]
+            }
+        },
+        "/identify": {
+            "post": {
+                "description": "Identify unknown faces from a person group.",
+                "operationId": "Face_Identify",
+                "parameters": [
+                    {
+                        "$ref": "../../Common/Parameters.json#/parameters/SubscriptionKey"
+                    },
+                    {
+                        "name": "body",
+                        "in": "body",
+                        "required": true,
+                        "x-ms-client-flatten": true,
+                        "schema": {
+                            "$ref": "#/definitions/IdentifyRequest"
+                        }
+                    }
+                ],
+                "consumes": [
+                    "application/json"
+                ],
+                "responses": {
+                    "200": {
+                        "description": "A successful call returns the identified candidate person(s) for each query face.",
+                        "schema": {
+                            "$ref": "#/definitions/IdentifyResult"
+                        },
+                        "examples": {
+                            "application/json": [
+                                {
+                                    "faceId": "c5c24a82-6845-4031-9d5d-978df9175426",
+                                    "candidates": [
+                                        {
+                                            "personId": "25985303-c537-4467-b41d-bdb45cd95ca1",
+                                            "confidence": 0.92
+                                        }
+                                    ]
+                                },
+                                {
+                                    "faceId": "65d083d4-9447-47d1-af30-b626144bf0fb",
+                                    "candidates": [
+                                        {
+                                            "personId": "2ae4935b-9659-44c3-977f-61fac20d0538",
+                                            "confidence": 0.89
+                                        }
+                                    ]
+                                }
+                            ]
+                        }
+                    },
+                    "default": {
+                        "description": "Error response.",
+                        "schema": {
+                            "$ref": "#/definitions/APIError"
+                        }
+                    }
+                },
+                "produces": [
+                    "application/json"
+                ]
+            }
+        },
+        "/verify?overload=persongroup": {
```

Comment:
Move overloaded operations to the `x-ms-paths` section, take a look in to the linter [errors](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/289380274#L675).

Reply:
You still need to move a [bunch more operations](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/297821831#L766-L839)

Reply:
Good, Looks like this comment is solved